modules/doc/content/newsletter/2023/2023_09.md

```diff
@@ -12,3 +12,44 @@ for a complete description of all MOOSE changes.
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+### `2023.09.06` Update
+
+- NetCDF upgrade to 4.9.2, now incorporating NetCDF via a git submodule
+- p-refinement can now be selectively disabled and reenabled on a
+  per-variable-group basis
+- New error indicator for Empirical Interpolation Method (EIM)
+  reduced-basis approximations
+- `Elem::quality(ASPECT_RATIO)` is now defined for all element types
+- Added option to use absolute instead of relative tolerances in
+  linear solvers
+- Added the capability to "upgrade" a vector's ghosting and/or
+  projection settings in subsequent `System::add_vector()` calls
+- Added `MeshRefinement::allow_unrefined_patches()` option
+- Refactored iterator declarations and added more range methods in
+  `MeshBase`: every filtered iterator that could be previously
+  accessed with a `...begin()` and `...end()` method now also has a
+  `...range()` method suitable for range-based for loops
+- Better PETSc versioning macros for use with pre-release PETSc
+  commits
+- Build system now sets `CFLAGS_OPT`, etc. environment variables, not
+  just `CXXFLAGS` and `CPPFLAGS` versions
+- Many more unit tests for ExodusII I/O and for element refinement
+  APIs
+- Assorted fixes 
+  - Fixed Exodus I/O of side sets on 2D triangles embedded in 3D
+    space
+  - Fixed Nemesis dimension variable output
+  - Fixed Nedelec one `FE` compatibility with `Tri7` and `Tet14`
+    geometric elements
+  - Fixed multiple issues with mesh refinement on meshes with certain
+    types of infinite elements
+  - Fixed no-parameters case and improved print statements in EIM code
+  - Fixed typos in code comments
+  - Minor fixes in vector FE example programs
+- Assorted optimizations
+  - an unnecessary pointer-indirection layer has been removed from
+    `DiffContext`
+  - local matrix allocation in `DiffContext` objects is now optional
+  - minor optimizations have been made to Hilbert-curve global
+    indexing calculations
```

Comment:
```suggestion
  - An unnecessary pointer-indirection layer has been removed from
    `DiffContext`
  - Local matrix allocation in `DiffContext` objects is now optional
  - Minor optimizations have been made to Hilbert-curve global
    indexing calculations
```

Capitalization consistency. Otherwise looks good. Thanks for the news update!